src/Pages/Board.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { MdModeEdit } from "react-icons/md";
+import styled from "styled-components";
+
+
+class Board extends React.Component {
+  render() {
+    const { data, componentKey } = this.props;
+    return (
+      <BoardComponent key={componentKey}  >
+        <div >
+            <Left src={data.pinImageUrl ? data.pinImageUrl[0] :""}/>
+            <div>
+                <Right src={data.pinImageUrl ? data.pinImageUrl[1] : ""}/>
+                <RightBottom src={data.pinImageUrl ? data.pinImageUrl[2] : ""}/>
+            </div>
```

Comment:
1. 삼항연산자로 잘 처리 해주셨는데 false 인 경우 깨진 이미지가 보이지 않을까요? 대체 이미지를 하나 정해두면 좋을거 같습니다.
2. 배열에도 구조분해 할당이 가능합니다.
```suggestion
const [pinImgFoo, pinImgBar, pinImgBaz] = this.props.data.pinImageUrl

            <Left src={pinImageUrl ? PinImgFoo :""}/>
            <div>
                <Right src={pinImageUrl ? pinImgBar : ""}/>
                <RightBottom src={pinImageUrl ? pinImgBaz : ""}/>
            </div>
```

Reply:
반영했습니다